### DISCUSSION TITLE
Firmware Update failed

### DISCUSSION THREAD

**Discussion by VaterMitBart** [NARUTOfzr/Neptune_3#65] [Q&A]
I tried to get the 1.0.5 beta on my Neptune 3 Pro. Now it says "Update Firmware" on the display for 30 minutes and nothing happens. Display version 1.4 is already on it

**Comment by Moody66** [comment]
Neptune 3 firmware on a Neptune 3 pro.... Ya Not going to work.

**Comment by Luk3hutchinson** [comment]
Your after this repository if your using the 3 Pro/Plus/Max 
[Pro/Plus/Max ](https://github.com/NARUTOfzr/Neptune_3_Pro_Plus_Max)
